controllers/teams.js

```diff
@@ -1058,6 +1060,11 @@ router.get('/:teamId/members', async (req, res, next) => {
 			} else {
 				actions = addButtonTrash(actions);
 			}
+			let userRolesContainsStudent;
+			if (FEATURE_STUDENTS_CANT_BE_TEAM_ADMINISTRATOR_ENABLED) {
+				const userRoles = roles.filter((r) => user.userId.roles.find((id) => id === r._id)).some(role => role.name === 'student');
+				userRolesContainsStudent = userRoles.some((role) => role.name === 'student');
```

Comment:
userRoles is already a boolean here, right?
```suggestion
				userRolesContainsStudent = roles.filter((r) => user.userId.roles.find((id) => id === r._id)).some(role => role.name === 'student');
```

Reply:
ah, you fixed it while I was typing :-)